daprdocs/content/en/developing-applications/building-blocks/service-invocation/service-invocation-overview.md

```diff
@@ -103,6 +103,10 @@ By default, all calls between applications are traced and metrics are gathered t
 
 The API for service invocation can be found in the [service invocation API reference]({{< ref service_invocation_api.md >}}) which describes how to invoke a method on another service.
 
+### gRPC proxying
+
+Dapr allows users to keep their own proto services and work natively with gRPC. This means there are no SDKs or custom gRPC services needed to have Dapr service invocation work with gRPC apps. For more information, see the [how-to tutotial for Dapr and gRPC]({{< ref howto-invoke-services-grpc.md >}}). 
```

Comment:
Can we make this sentence clearer
'This means there are no SDKs or custom gRPC services needed to have Dapr service invocation work with gRPC apps."

How about
"This means that you can use service invocation to call your existing gRPC apps without having to include any Dapr SDKs or include custom gRPC services."

Reply:
Changed.